# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../bincimap12/glibc-2_10.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur bincimap-1.2.13final.orig/src/convert.h bincimap-1.2.13final/src/convert.h
--- bincimap-1.2.13final.orig/src/convert.h	2005-02-08 22:30:22.000000000 +0200
+++ bincimap-1.2.13final/src/convert.h	2009-08-08 10:36:12.000000000 +0300
@@ -112,12 +112,12 @@
       unsigned char d = *(i + 1);
       
       char *t;
-      if ((t = strchr(hexchars, c)) == 0)
+      if ((t = (char *) strchr(hexchars, c)) == 0)
 	return "out of range";
       n = (t - hexchars) << 4;
       
       
-      if ((t = strchr(hexchars, d)) == 0)
+      if ((t = (char *) strchr(hexchars, d)) == 0)
 	return "out of range";
       n += (t - hexchars);

